test-integration/jdl-samples/ms-mf-react-eureka-oauth2-mariadb-infinispan/blog-store.jdl

```diff
@@ -27,7 +27,7 @@ application {
     creationTimestamp 1617901618886
     jwtSecretKey "ZjY4MTM4YjI5YzMwZjhjYjI2OTNkNTRjMWQ5Y2Q0Y2YwOWNmZTE2NzRmYzU3NTMwM2NjOTE3MTllOTM3MWRkMzcyYTljMjVmNmQ0Y2MxOTUzODc0MDhhMTlkMDIxMzI2YzQzZDM2ZDE3MmQ3NjVkODk3OTVmYzljYTQyZDNmMTQ="
     packageName com.okta.developer.gateway
-    prodDatabaseType mariadb
+    prodDatabaseType postgresql
```

Comment:
Shouldn't we replace with MySQL since that's a closer dialect? Also, the sample name should change since it has "mariadb" in it.

Reply:
MySQL uses mariadb driver, the problem would persist.

Reply:
This sample is still focused on mariadb.
We should revert to mariadb once the issue is fixed and non-reactive samples are still using mariadb.